src/components/plots/xyPlot.ts

```diff
@@ -71,6 +71,49 @@ module Plottable {
       return this;
     }
 
+    protected _updateExtentsForAttr(attr: string) {
+      if (attr === "x") {
+        return super._updateExtentsForAttr(attr, this._xFilter());
+      } else if (attr === "y") {
+        return super._updateExtentsForAttr(attr, this._yFilter());
+      }
+      return super._updateExtentsForAttr(attr);
+    }
```

Comment:
The map of filters is kind of annoying because the filter now has to look up the scale each time it's executed:
```Typescript
var filterByX = (datum: any, index: number, dataset: Dataset, metadata: Plots.PlotMetadata) => {
  var xBinding = this._attrBindings.get("x");
  if (xBinding != null) {
    var xAccessor = xBinding.accessor;
    var xScale = xBinding.scale;
    if (xScale != null) {
      var range = xScale.range();
      return Utils.Methods.inRange(xScale.scale(xAccessor(d, i, dataset, metadata)), range[0], range[1]);
    }
  }
  return true;
}
```
This is because at the time `automaticallyAdjustYScaleOverVisiblePoints(true) is called, the projectors may not have been set up correctly.